rskj-core/src/main/java/co/rsk/net/handler/TxValidator.java

```diff
@@ -50,7 +47,6 @@
     private final Blockchain blockchain;
 
     private final List<TxValidatorStep> validatorSteps = new LinkedList<>();
-    private final List<TxFilter> txFilters = new LinkedList<>();
```

Comment:
Was this unused? Do we know why/if it should actually have been used?

Reply:
This was unused for sure (it was reported by Sonar). About how this should have been used, maybe @donequis could tell us more

Reply:
It's fine. We're tracking this issue and fixing it as part of this sprint.